kiali-operator/templates/deployment.yaml

```
@@ -109,6 +110,8 @@ spec:
         {{- else }}
           value: "/etc/ansible/ansible.cfg"
         {{- end }}
+        - name: ANSIBLE_LOCAL_TEMP
+          value: "/tmp/ansible/tmp"
```

Add ANSIBLE_REMOTE_TEMP also:
```suggestion
        - name: ANSIBLE_LOCAL_TEMP
          value: "/tmp/ansible/tmp"
        - name: ANSIBLE_REMOTE_TEMP
          value: "/tmp/ansible/tmp"
```